Equilobe.DailyReport.BL/BitBucket/BitBucketApiUrls.cs

```diff
@@ -26,5 +26,10 @@ public static string Contributors(string team, int page)
         {
             return string.Format("teams/{0}/members?page={1}", team, page);
         }
+
+        public static string CommitDiffs(string owner, string repository, string commitHash)
+        {
```

Comment:
We should do the same thing for github. Let's see what @DanBadea @ccriveteanu think about this.

Reply:
I created another task for this in Jira.